crates/testing-framework/src/asserts/mod.rs

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2022 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+pub(crate) mod parser;
+
+use crate::services::JValue;
+
+/// Assert language structure: Assert.
```

Comment:
This comment confuses me. I dont understant its meaning and the duplication of the word assert.

Reply:
Well, it's remnants of my previous ideas on testing framework.